50a3c9bc6ddf2cefb679425097cf5c79f9d4e4ff
diff --git a/trunk/PKGBUILD b/trunk/PKGBUILD
index f86120baf4013ba168df44a7130bd9a98f37942d..bb4e0b03197303a8ce1ce26cbe0bab8716d21d72 100644
--- a/trunk/PKGBUILD
+++ b/trunk/PKGBUILD
@@ -3,7 +3,7 @@
 # Contributor: Vesa Kaihlavirta <vegai@iki.fi>
 
 pkgbase=quassel
-pkgname=('quassel-core' 'quassel-client' 'quassel-client-small' 'quassel-monolithic')
+pkgname=('quassel-client-custom')
 pkgver=0.12.4
 pkgrel=3
 pkgdesc="Next-generation distributed IRC client"
@@ -16,15 +16,18 @@ makedepends=('qt5-base' 'qt5-tools' 'qt5-webkit' 'qca-qt5' 'qt5-script'
 source=(http://quassel-irc.org/pub/$pkgbase-$pkgver.tar.bz2
         0001-Fix-handling-systray-notifications.patch
         $pkgbase.service
-        $pkgbase.conf)
+        $pkgbase.conf
+        custom.patch)
 sha256sums=('93e4e54cb3743cbe2e5684c2fcba94fd2bc2cd739f7672dee14341b49c29444d'
             '5a465f33c5f233983a12139a1a121665c4ddab9e205997daa453dcffd255156a'
             '5dbe20290f3361b9b7a74a52905137e76b656976febf2d31082a2276f9dcde7f'
-            'f3031ea8217e01ba42cea14606169e3e27affa5918968ffd5a03c21ae92fe2b8')
+            'f3031ea8217e01ba42cea14606169e3e27affa5918968ffd5a03c21ae92fe2b8'
+            'e8fa1969e66bdb0a4127c8065b02340e5fb39b626089ebdeeb05991478a575fa')
 
 prepare() {
   cd "${srcdir}"
   mkdir build-{core,client,client-small,mono}
+  patch -d "quassel-$pkgver" -N -p0 <custom.patch
   cd quassel-${pkgver}
 
   # Fix handling systray notifications
@@ -46,28 +49,10 @@ _build() (
 )
 
 build() {
-  _build core \
-    -DWANT_QTCLIENT=OFF \
-    -DWANT_MONO=OFF \
-    -DWITH_OXYGEN=OFF \
-    -DWITH_WEBKIT=OFF
-
   _build client \
     -DWANT_CORE=OFF \
     -DWANT_MONO=OFF \
     -DWITH_KDE=ON
-
-  _build client-small \
-    -DWANT_CORE=OFF \
-    -DWANT_MONO=OFF \
-    -DWITH_WEBKIT=OFF \
-    -DCMAKE_DISABLE_FIND_PACKAGE_dbusmenu-qt5=ON \
-    -DCMAKE_DISABLE_FIND_PACKAGE_Phonon4Qt5=ON
-
-  _build mono \
-    -DWANT_CORE=OFF \
-    -DWANT_QTCLIENT=OFF \
-    -DWITH_KDE=ON
 }
 
 package_quassel-core() {
@@ -89,10 +74,11 @@ install=$pkgbase.install
     "${pkgdir}"/etc/conf.d/$pkgbase
 }
 
-package_quassel-client() {
+package_quassel-client-custom() {
 pkgdesc="Next-generation distributed IRC client - client only"
 depends=('qt5-base' 'qt5-webkit' 'knotifyconfig' 'hicolor-icon-theme')
-conflicts=('quassel-monolithic' 'quassel-client-small')
+provides=('quassel-client')
+conflicts=('quassel-monolithic' 'quassel-client-small' 'quassel-client')
   cd "${srcdir}"/build-client
 
   make DESTDIR="${pkgdir}" install
diff --git a/trunk/custom.patch b/trunk/custom.patch
new file mode 100644
index 0000000000000000000000000000000000000000..50d944014535b580e3397faec416d4d933131a7f
--- /dev/null
+++ b/trunk/custom.patch
@@ -0,0 +1,10 @@
+--- src/uisupport/uistyle.cpp
++++ src/uisupport/uistyle.cpp
+@@ -869,6 +869,7 @@
+             nick.chop(chopCount);
+     }
+     quint16 hash = qChecksum(nick.toLatin1().data(), nick.toLatin1().size());
++    hash ^= hash >> 8; hash ^= hash >> 3;
+     return (_senderHash = (hash & 0xf) + 1);
+ }
+ 
