--- /etc/pacman.conf	2014-10-06 22:07:27.326802722 -0400
+++ /etc/pacman.conf	2014-10-13 21:14:15.724438001 -0400
@@ -30,7 +30,7 @@
 
 # Misc options
 #UseSyslog
-#Color
+Color
 #TotalDownload
 CheckSpace
 #VerbosePkgLists
@@ -89,11 +89,20 @@
 #[multilib-testing]
 #Include = /etc/pacman.d/mirrorlist
 
-#[multilib]
-#Include = /etc/pacman.d/mirrorlist
+[multilib]
+Include = /etc/pacman.d/mirrorlist
 
 # An example of a custom package repository.  See the pacman manpage for
 # tips on creating your own repositories.
 #[custom]
 #SigLevel = Optional TrustAll
 #Server = file:///home/custompkgs
+
+[infinality-bundle]
+Server = http://bohoomil.com/repo/$arch
+
+[infinality-bundle-fonts]
+Server = http://bohoomil.com/repo/fonts
+
+[infinality-bundle-multilib]
+Server = http://bohoomil.com/repo/multilib/$arch
