--- /etc/fonts/conf.avail.infinality/35-repl-custom.conf	2014-10-07 16:36:00.496040753 -0400
+++ /etc/fonts/conf.avail.infinality/35-repl-custom.conf	2014-10-07 16:56:56.816021332 -0400
@@ -40,24 +40,20 @@
   -->
 
   <!-- ## sans-serif ## -->
-  <!--
   <alias>
     <family>sans-serif</family>
     <prefer>
-      <family>DEFAULT SANS-SERIF FACE</family>
+      <family>Lato</family>
     </prefer>
   </alias>
-  -->
 
   <!-- ## monospace ## -->
-  <!--
   <alias>
     <family>monospace</family>
     <prefer>
-      <family>DEFAULT MONOSPACE FACE</family>
+      <family>Inconsolata</family>
     </prefer>
   </alias>
-  -->
 
   <!-- ## fantasy ## -->
   <!--
